components/ActivityIndicator/examples/supplemental/ActivityIndicatorExampleSupplemental.m

```diff
@@ -122,8 +121,8 @@ - (void)didChangeSliderValue:(UISlider *)slider {
   self.activityIndicator1.progress = slider.value;
   self.activityIndicator2.progress = slider.value;
   self.activityIndicator3.progress = slider.value;
-  UITableViewCell *cell =
-      [self.tableView cellForRowAtIndexPath:[NSIndexPath indexPathForRow:1 inSection:0]];
+  UITableViewCell *cell = [self.tableView cellForRowAtIndexPath:[NSIndexPath indexPathForRow:1
+                                                                                   inSection:0]];
```

Comment:
I would argue that the previous formatting is better.
Prefer to break assignment to maintain a single line invocation.

Reply:
I actually kind of like this because it makes the parameters to the outer method more prominent. That being said, I don't have a strong leaning either way.